internal/node/component/info/network_config.go

```diff
@@ -217,8 +226,10 @@ func customWorkerConfig(worker worker.Worker, network network.Source, parameters
 	return config
 }
 
-func getEndpointConfig() Endpoint {
-	return Endpoint{
+const mastodonInstanceDescription = "A Mastodon instance is required. Please follow <a href=\"https://github.com/RSS3-Network/Mastodon-Instance-Kit\" target=\"_blank\">the guide</a> to deploy your own instance. Once deployed, set the endpoint of your Mastodon instance here."
```

Comment:
"A Mastodon instance is required. Please follow <a href=\"https://github.com/RSS3-Network/Mastodon-Instance-Kit\" target=\"_blank\">the guide</a> to either deploy a new Mastodon instance or modify an existing Mastodon instance. After completing either option, enter your Mastodon endpoint (format: your_instance_ip:9092) here."